playground-app/src/App.tsx

```diff
@@ -4,6 +4,18 @@ import { FlagdCore, MemoryStorage } from "@openfeature/flagd-core";
 import { ScenarioName, scenarios } from "./scenarios";
 import type { FlagValueType } from "@openfeature/core";
 import { getString, isValidJson } from "./utils";
+import { Editor } from "@monaco-editor/react";
+import { Observable } from "react-use/lib/useObservable";
+
+declare global {
+  var component$: Observable<{ ref: HTMLElement}>;
+}
+
+// see: https://github.com/squidfunk/mkdocs-material/discussions/3429
+const BODY_COLOR_SCHEME_ATTR = 'data-md-color-scheme';
+const PALETTE_SWITCH_SELECTOR = '[data-md-component=palette]';
+const getPalette = () => document.body.getAttribute(BODY_COLOR_SCHEME_ATTR) &&
+  document.body.getAttribute(BODY_COLOR_SCHEME_ATTR) !== 'default' ? 'custom-dark' : 'custom';
```

Comment:
This is a bit hacky, but the mkdocs author recommends it: https://github.com/squidfunk/mkdocs-material/discussions/3429